### PR TITLE
OSIDB-3394: Update OpenAPI schema to include new Flaw sources

### PR DESCRIPTION
**Pull request by MrMarble**
# OSIDB-3394: Update OpenAPI schema to include new Flaw sources

## Checklist:

- [x] Commits consolidated
- [x] Changelog updated
- [x] Test cases added/updated
- [x] Jira ticket updated

## Summary:

Updated OpenAPI schema to include new `CVEORG` Flaw source

## Considerations:

Closes OSIDB-3394

### CODE REVIEW

File: src/types/zodFlaw.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { zodOsimDateTime, ImpactEnumWithBlank, ZodFlawClassification, ZodAlertSc
 import { ZodAffectSchema, type ZodAffectType } from './zodAffect';
 
 export const RequiresDescriptionEnumWithBlank = { '': '', ...RequiresCveDescriptionEnum } as const;
-export const Source642EnumWithBlank = { '': '', ...Source642Enum } as const;
```
Comment (C-Valen) [inline src/types/zodFlaw.ts:16]: Shouldn't this be changed from `Source642EnumWithBlank` to `Source521EnumWithBlank`

**Review by C-Valen — COMMENTED** [review]
Quick question, there is no need to update the dropdowns from FlawForm/ Advanced search?

File: src/types/zodFlaw.ts (hunk 1 above)
Comment (MrMarble) [reply src/types/zodFlaw.ts:16]: Since the name of the enum changes each time, I wanted to do the minimum changes but I'll update it for consistency

Comment (MrMarble) [reply src/types/zodFlaw.ts:16]: As for the dropdowns, this new source is read only, so the users should not be able to manually create a flaw with it. The Advanced search drop down uses this enum directly, so it is already updated

**Review by C-Valen — APPROVED** [review]

File: src/types/zodFlaw.ts (hunk 1 above)
Comment (jobselko) [reply src/types/zodFlaw.ts:16]: Yeah, that changing enum name is a bug in OSIDB :/

**Pull request merged**